apps/site/components/Downloads/MinorReleasesTable/index.tsx

```diff
@@ -15,50 +19,81 @@ type MinorReleasesTableProps = {
   releases: Array<MinorVersion>;
 };
 
-export const MinorReleasesTable: FC<MinorReleasesTableProps> = ({
-  releases,
-}) => {
-  const t = useTranslations('components.minorReleasesTable');
+const MinorReleasesTable: FC<MinorReleasesTableProps> = ({ releases }) => {
+  const t = useTranslations('components');
```

Comment:
```suggestion
  const t = useTranslations();
```

No usage of namespaces 👀